examples/sdk-usage/src/main/java/io/opentelemetry/sdk/example/BasicExample.java

```diff
@@ -0,0 +1,44 @@
+package io.opentelemetry.sdk.example;
+
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.TracerSdk;
+import io.opentelemetry.sdk.trace.TracerSdkFactory;
+
+/**
+ * This example shows how to instantiate an OpenTelemetry tracer.
+ */
+class BasicExample {
```

Comment:
This example is really an example of API usage, just using the SDK classes. I think these examples should focus on operator/application owner setup/configuration of the SDK, rather than repeating API usage examples.

Reply:
isn't that covered by [ConfigureSpanProcessorExample](https://github.com/dynatrace-oss-contrib/opentelemetry-java/blob/sdk-usage/examples/sdk-usage/src/main/java/io/opentelemetry/sdk/example/ConfigureSpanProcessorExample.java) and [ConfigureTraceExample](https://github.com/dynatrace-oss-contrib/opentelemetry-java/blob/sdk-usage/examples/sdk-usage/src/main/java/io/opentelemetry/sdk/example/ConfigureTraceExample.java)?